src/storage/disk_manager.c

```diff
@@ -905,8 +906,28 @@ disk_set_creation (THREAD_ENTRY * thread_p, INT16 volid, const char *new_vol_ful
     }
 
   /* Modify volume creation information */
-  memcpy (&vhdr->db_creation, new_dbcreation, sizeof (*new_dbcreation));
-  memcpy (&vhdr->chkpt_lsa, new_chkptlsa, sizeof (*new_chkptlsa));
+
+  /* TODO #
+   * Currently, the volume creation time and database creation time are updated only when the new database file is created.
+   * However, the existing code performs a memcpy even when the new_dbcreation value is the same as vhdr->db_creation, regardless of whether the database is being created.
+   * For now, the code has been changed as below, but it seems necessary to determine whether to refactor and improve this in the future.
+   */
+  if (is_new_db_creation == true)
+    {
+      assert (*new_dbcreation != vhdr->db_creation_time);
+      vhdr->vol_creation_time = time (NULL);
+      memcpy (&vhdr->db_creation_time, new_dbcreation, sizeof (*new_dbcreation));
+    }
+  else
+    {
+      assert (*new_dbcreation == vhdr->db_creation_time);
+    }
```

Comment:
NIT:

일관되게 `vhdr->db_creation_time`를 기준으로 비교될 수 있도록 다음과 같이 코드를 정리하면 좋겠네요.
```suggestion
  if (is_new_db_creation == true)
    {
      assert (vhdr->db_creation_time != *new_dbcreation);
      vhdr->vol_creation_time = time (NULL);
      memcpy (&vhdr->db_creation_time, new_dbcreation, sizeof (*new_dbcreation));
    }
  else
    {
      assert (vhdr->db_creation_time == *new_dbcreation);
    }
```

Reply:
수정하였습니다.